src/Sylius/Behat/Context/Api/Admin/ManagingPaymentsContext.php

```diff
@@ -44,6 +47,34 @@ public function iAmBrowsingPayments(): void
         $this->client->index(Resources::PAYMENTS);
     }
 
+    /**
+     * @When I go to the details of the first payment's order
+     */
+    public function iGoToTheDetailsOfTheFirstPaymentSOrder(): void
+    {
+        $firstPayment = $this->responseChecker->getCollection($this->client->getLastResponse())[0];
+
+        /** @var OrderInterface $order */
+        $order = $this->iriConverter->getResourceFromIri($firstPayment['order']);
+
+        $this->client->customItemAction(Resources::ORDERS, $order->getTokenValue(), HttpRequest::METHOD_GET, 'payments');
+    }
+
+    /**
+     * @Then I should see the details of order :order
+     */
+    public function iShouldSeeOrderWithDetails(OrderInterface $order): void
+    {
+        Assert::true(
+            $this->responseChecker->hasItemWithValue(
+            $this->client->getLastResponse(),
+            'order',
+            $this->sectionAwareIriConverter->getIriFromResourceInSection($order, 'admin'),
+        ),
+            sprintf('Order with number %s does not exist', $order->getNumber()),
+        );
```

Comment:
The formatting here is weird, but let's leave it for the ecs build to take care of.